package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@api3/chains",
+  "name": "@phase21/chains",
   "license": "MIT",
-  "version": "4.2.0",
+  "version": "0.1.1",
```

Comment:
And reverting this to 0.1.1 on `main`?

Reply:
This package version is for `@phase21/chains` package. You can consider this branch will not be merged any time soon as it will be for test purposes only.